pkg/oomstore/revision.go

```diff
@@ -20,3 +26,33 @@ func (s *OomStore) GetRevision(ctx context.Context, id int) (*types.Revision, er
 func (s *OomStore) GetRevisionBy(ctx context.Context, groupID int, revision int64) (*types.Revision, error) {
 	return s.metadata.GetRevisionBy(ctx, groupID, revision)
 }
+
+func (s *OomStore) createDummySnapshotTable(ctx context.Context, revision *types.Revision) error {
```

Comment:
prefer "genesis" or "sentinel" or "first" over "dummy" to show "this is the first snapshot table"